dashboard/app/views/layouts/application.html.haml

```diff
@@ -14,6 +14,10 @@
     = javascript_include_tag "js/#{js_locale}/common_locale"
     %script{src: minifiable_asset_path('js/essential.js')}
     %script{src: minifiable_asset_path('js/code-studio-common.js')}
+
+    //Included early to prevent JS Errors from being sent to New Relic
+    %script{src: minifiable_asset_path('js/layouts/application.js')}
+
```

Comment:
Some errors (e.g. `UncaughtException: duplicate formal argument e` in very old versions of Firefox) appear to be coming from files we've imported but haven't run yet.  Maybe there's a better workaround?